### ISSUE TITLE
Using wrapAngularDevkitSchematic in a workspace-schematic converts the workspace to v1

### ISSUE THREAD

**Issue by beeman** [nrwl/nx#4417]
## Current Behavior

When creating a workspace-generator that uses `wrapAngularDevkitSchematic`, the project type gets set to 1, whereas running that schematic manually keeps it at 2.

## Expected Behavior

The project type should stay at 2.

## Steps to Reproduce

This is the project I created, you can find it [here](https://github.com/beeman/workspace-generator-issue):

```
yarn create nx-workspace workspace-generator-issue --preset=empty --package-manager=yarn
cd workspace-generator-issue
yarn add -D @nrwl/react-native
nx g @nrwl/react-native:app native
```

When creating a library manually, it all goes well as you can see [here](https://github.com/beeman/workspace-generator-issue/compare/manual?expand=1#diff-8615bf67663730c68237b51aa01c74d409e3ebfa0ddb23cde704c9a2e97779fdR78):

```
nx g @nrwl/react-native:lib post --directory native --tags scope:native
```

Now, we create a `workspace-generator` that allows to create these libs:

```
nx generate @nrwl/workspace:workspace-generator native-lib
``` 

Update the contents of `tools/generators/native-lib/index.ts` to the following:

```typescript
import { formatFiles, installPackagesTask, Tree } from '@nrwl/devkit';
import { wrapAngularDevkitSchematic } from '@nrwl/tao/src/commands/ngcli-adapter';

export default async function (host: Tree, schema: any) {
  const generator = wrapAngularDevkitSchematic('@nrwl/react-native', 'library');

  await generator(host, {
    name: schema.name,
    directory: 'native',
    tags: 'scope:native',
  });

  await formatFiles(host);
  return () => {
    installPackagesTask(host);
  };
}
```

And run it:
```
nx workspace-generator native-lib post
```

This will convert the workspace file to v1 as you can [see here](https://github.com/beeman/workspace-generator-issue/compare/generator?expand=1#diff-8615bf67663730c68237b51aa01c74d409e3ebfa0ddb23cde704c9a2e97779fdR2).

### Failure Logs

No errors

### Environment

```
  Node : 14.13.1
  OS   : darwin x64
  yarn : 1.22.5

  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.0.20
  @nrwl/cypress : 10.4.4
  @nrwl/devkit : 11.0.20
  @nrwl/eslint-plugin-nx : 11.0.20
  @nrwl/express : Not Found
  @nrwl/jest : 10.4.4
  @nrwl/linter : 10.4.4
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : 10.4.4
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.20
  @nrwl/web : 10.4.4
  @nrwl/workspace : 11.0.20
  typescript : 4.0.5
```

**Comment by vsavkin** [comment]
This is fixed here: 0c0a29da71436aed8d3b1771b64dd6cb1cc4c623

Thank you!

**Comment by beeman** [comment]
Thanks for the quick fix Victor!
